packages/react-instantsearch-core/src/core/translatable.js

```diff
@@ -23,7 +22,7 @@ export default function translatable(defaultTranslations) {
         const { translations } = this.props;
 
         const translation =
-          translations && has(translations, key)
+          translations && translations.hasOwnProperty(key)
```

Comment:
So, should this be reverted to a regular boolean check  (which technically would be breaking) or not?

Reply:
I would say _yes_.

Reply:
Nope, not this one.

Reply:
True!

Reply:
perfect, so I'll leave this as-is